linode/manager
# change: [M3-7785] - Improve Dev Tools

## Description 📝
This PR brings non-breaking, minor improvement to the dev tools.

## Changes  🔄
- Add a theme select to persist theme choice while using MSW
- Add a rest button to bring the feature flags values back to their Launch Darkly defaults
- Color the dev tools icon red when MSW is enabled
- Add a link to the preference modal for convenience
- Comment out banners present of EVERY page cluttering the UI. (they can be easily re-enabled in `serverHandlers.ts`
- Update documentation

## Preview 📷
![Screenshot 2024-02-22 at 11 49 19](https://github.com/linode/manager/assets/130582365/27cf4bff-f1e0-4761-b2ba-34dcd4d25b37)

## How to test 🧪

### Verification steps
- Pull local code locally and `yarn up` your application
- Confirm behavior of dev tools menu

## As an Author I have considered 🤔

*Check all that apply*
